File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/declare/GraduationDesignDeclareBean.java
package top.zbeboy.isy.web.bean.graduate.design.declare;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.GraduationDesignDeclare;

/**
 * Created by zbeboy on 2017/6/8.
 */
@EqualsAndHashCode(callSuper = false)
@Data
public class GraduationDesignDeclareBean extends GraduationDesignDeclare {
    private String staffName;
    private String academicTitleName;
    private String studentName;
    private String studentNumber;
    private String organizeName;
    private String subjectTypeName;
    private String originTypeName;
    private String presubjectTitle;
    private int staffId;
    private int studentId;
    private String graduationDesignReleaseId;
    private int publicLevel;
    private String scoreTypeName;
    private String defenseOrderId;
}

File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/declare/GraduationDesignDeclareBean.kt
package top.zbeboy.isy.web.bean.graduate.design.declare

import top.zbeboy.isy.domain.tables.pojos.GraduationDesignDeclare

/**
 * Created by zbeboy 2018-01-26 .
 **/
class GraduationDesignDeclareBean : GraduationDesignDeclare() {
    var staffName: String? = null
    var academicTitleName: String? = null
    var studentName: String? = null
    var studentNumber: String? = null
    var organizeName: String? = null
    var subjectTypeName: String? = null
    var originTypeName: String? = null
    var presubjectTitle: String? = null
    var staffId: Int = 0
    var studentId: Int = 0
    var graduationDesignReleaseId: String? = null
    var publicLevel: Int = 0
    var scoreTypeName: String? = null
    var defenseOrderId: String? = null
}